### DISCUSSION TITLE
virus blocking msi download

### DISCUSSION THREAD

**Discussion by franskaisson** [porrey/Virtual-ZPL-Printer#57] [General]
Trying to download the latest msi on Windows 11. Windows Defender blocks the download with a virus warning. Inspecting the warning shows that trojan Trojan:Win32/MalUri.A!cl was identified.
Doesn't sound good. Is this a known issue?

**Comment by porrey** [comment]
Could be a false positive. Please check the file with online scanners to be sure. I will check the file as well.

**Reply by franskaisson** [reply]
The link to download the latest msi (https://github.com/porrey/Virtual-ZPL-Printer/raw/main/Installer/Virtual%20ZPL%20Printer%20Setup.msi) now gives a 404...

**Reply by pandente** [reply]
Hi, i face the same issue. Just tried to download it, but 404 error

thanks

**Comment by porrey** [comment]
I deleted it for now until I can confirm if it is an issue or not.

**Comment by porrey** [comment]
I rebuilt the setup and uploaded the new version. I scanned the file with multiple apps and nothings reports it as a threat. I also took my PC offline and scanned the drive from another computer and it was clean. A full scan with Windows Defender does not find any threats. However, when downloading the file Windows Defender claims it contains a threat. This does appear to be a false positive.

**Comment by franskaisson** [comment]
The warning changed however: "Trojan:Script/Wacatac.H!ml"

**Comment by porrey** [comment]
I installed a clean version of Windows 11 on my Mac under Parallels and downloaded the file, scanned it and did not get any threat notifications. I also cloned the repo and scanned that and had the same result, no threats (I used Microsoft Defender). I rebuilt the MSI in the clean Windows environment and uploaded it as a precaution.

**Reply by franskaisson** [reply]
ok thank you.
running on windows 11 with the latest defender updates still gives the virus warning when downloading.
Also if I disable Defender and download the msi file and then scan it while it is on my disc I also get the virus warning and defender removes the file.
I am not running on mac however..

**Reply by franskaisson** [reply]
I am now also 100 sure that it is Windows Defender giving a false positive.

**Comment by porrey** [comment]
I uploaded the file to Microsoft for scanning and no malware was found.
